README.md

```diff
@@ -210,6 +210,25 @@ Transforms a DateTime into an arbitrarily formatted string:
 {{ date | dateTimeToFormat:'MMMM d, yyyy' }}
 ```
 
+#### `dateTimeToLocaleString`
+
+Transforms a DateTime into an human-readable, internationalized string:
+
+```
+{{ date | dateTimeToLocaleString:DateTime.DATETIME_FULL }}
+
+{{ date | dateTimeToLocaleString:{
+                                    year: 'numeric',
+                                    month: 'long',
+                                    day: 'numeric',
+                                    hour: 'numeric',
+                                    minute: '2-digit',
+                                    timeZoneName: 'short'
+                                  } }}
+```
+
+The pipe takes [`Intl.DateTimeFormat`][intl-datetimeformat] as the first parameter and Luxon´s [`LocaleOptions`][luxon-localeoptions] as an optional second parameter to override the configuration options on this DateTime.
```

Comment:
None of the other pipes take `opts` right now, so let’s leave that off for now and add it to all pipes in a separate major change

Reply:
Sure, updated the PR.